integration/pkg/aws/aws.go

```diff
@@ -0,0 +1,146 @@
+package aws
+
+import (
+	"context"
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2Types "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/aws/aws-sdk-go-v2/service/networkfirewall"
+	nfwTypes "github.com/aws/aws-sdk-go-v2/service/networkfirewall/types"
+	tags "github.com/aws/aws-sdk-go-v2/service/resourcegroupstaggingapi"
+)
+
+const (
+	// These are temporary because CIDR math is hard
```

Comment:
Using a `/24` for each subnet should be enough and easy math 😄

Reply:
At one point I was going to support a `--cidr` flag, but gave up after it wasn't super straightforward to do CIDR math in Go haha so I just made them consts. Good point that we can size this down to a `/24` though.